app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self  # Add the person to the people dictionary
+
+    def __repr__(self) -> str:
+        return f"Person(name={self.name}, age={self.age})"
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person_data in people:
+        person_instance = Person(name=person_data["name"],
+                                 age=person_data["age"])
+        person_list.append(person_instance)
```

Comment:
you can use list comprehension here

Reply:
agree, @94nj111 next time please reject if you have comments

Reply:
list comprehension would be more readable as it's just list creating